src/server.js

```diff
@@ -271,6 +267,15 @@ function ldapAuth(client, user, password, callback) {
 	var userDN = user.replace(/([,\\\/#+<>;"= ])/g, "\\$1");
 	var bindDN = Helper.config.ldap.primaryKey + "=" + userDN + "," + Helper.config.ldap.baseDN;
 
+	var ldapclient = ldap.createClient({
```

Comment:
from what I can tell, createClient() will be called regardless if LDAP is enabled